src/py/py_multipatch.cpp

```diff
@@ -833,13 +844,12 @@ int AddBoundariesFromContinuity(const py::list& boundary_splines,
   };
 
   // Tangential Vector on boundary based on its derivative
-  auto tangential_vector = [&cpp_spline_list, &para_dim_, &dim_](
-                               const int& patch_id,
-                               const int& face_id) -> std::vector<double> {
+  auto normal_vector = [&cpp_spline_list, &para_dim_, &dim_](
+                           const int& patch_id,
+                           const int& face_id) -> std::vector<double> {
     // init return value (are default initialized to 0)
     std::vector<double> para_coord(para_dim_), bounds(2 * para_dim_),
-        tangential_vector(dim_);
-    std::vector<int> orders(para_dim_);
+        normal_vector(dim_), jacobian(dim_ * para_dim_);
```

Comment:
Based on no warning/error, I guess this doesn't necessarily count as "over shadowing" variable?